server/src/main/java/com/cloud/storage/StorageManagerImpl.java

```diff
@@ -776,7 +776,7 @@ public PrimaryDataStoreInfo createPool(CreateStoragePoolCmd cmd) throws Resource
                 throw new InvalidParameterValueException("Missing parameter hypervisor. Hypervisor type is required to create zone wide primary storage.");
             }
             if (hypervisorType != HypervisorType.KVM && hypervisorType != HypervisorType.VMware && hypervisorType != HypervisorType.Hyperv && hypervisorType != HypervisorType.LXC
-                    && hypervisorType != HypervisorType.Any) {
+                    && hypervisorType != HypervisorType.Any && hypervisorType != HypervisorType.Simulator) {
```

Comment:
shouldn't `Any` already include `Simulator` somehow?

Reply:
background, there is too much mention of simulator in production code. If we can avoid this we should.

Reply:
Ideally, but I think the current meaning of `Any` throughout the code may not include simulator